contributors/design-proposals/standardization-of-development-process.md

```diff
@@ -0,0 +1,356 @@
+used throughout the process of proposing an enhancement, scoping its design,
+tracking its implementation, and agreeing on criteria for graduation to general
+availability.
+
+As the local bodies of governance SIGs should have broad latitude in describing
+what constitutes an enhancement which should be tracked through the KEP process;
+hopefully the desire to be included in the rich narrative of a release announcement
+will encourage broad participation in the KEP process across SIGs. SIGs should
+also have the freedom to customize the KEP template according to their SIG
+specific concerns. For example the KEP template used to track API changes will
+likely have different subsections than the template for proposing governance
+changes.
+
+### KEP Template
+
+The template for a KEP will be submitted in a separate proposal
+
+### KEP Workflow
+
+A KEP is proposed to have the following states
+
+- **opened**: a new KEP has been filed but not triaged by the responsible SIG or
+  working group
+- **accepted**: the motivation has been accepted by the SIG or working group as in
+  road map
+- **scoped**: the design has been approved by the SIG or working group
+- **started**: the implementation of the KEP has begun
+- **implemented**: the implementation of the KEP is complete
+- **deferred**: the KEP has been postponed by the SIG or working group despite
+  agreement on the motivation
+- **superseded**: the KEP has been superseded by another KEP
+- **retired**: the implementation of the KEP has been removed
+- **rejected**: the KEP has been rejected by the SIG or working group
+- **orphaned**: the author or developer of the KEP is no longer willing or able
+  to complete implementation
+
+with possible paths through the state space
+
+- opened -> deferred (a)
+- opened -> rejected (b)
+- opened -> orphaned (c) 
+- opened -> accepted -> orphaned (d)
+- opened -> accepted -> scoped -> superseded (e)
+- opened -> accepted -> scoped -> orphaned (f) 
+- opened -> accepted -> scoped -> started -> retired (g)
+- opened -> accepted -> scoped -> started -> orphaned (h)
+- opened -> accepted -> scoped -> started -> superseded (i)
+- opened -> accepted -> scoped -> started -> implemented (j)
+- opened -> accepted -> scoped -> started -> implemented -> retired (k)
+
+the happy path is denoted by (j) where an KEP is opened; accepted by a SIG as in
+their roadmap; fleshed out with a design; started; and finally implemented. As
+Kubernetes continues to mature, hopefully metrics on the utilization of features
+will drive decisions on what features to maintain and which to deprecate and so
+it is possible that a KEP would be retired if its functionality no longer provides
+sufficient value to the community.
+
+### Git and GitHub Implementation
+
+Practically an KEP would be implemented as a pull request to a central repository
+with the following example structure
+
+```
+├── 0000-kep-template.md
+├── CODEOWNERS
+├── index.md
+├── sig-architecture
+│   ├── deferred
```

Comment:
This seems like it would require move movement of documents than is ideal. I could imagine distinguishing implemented from unimplemented proposals.

Reply:
it may require more movement than is ideal but I think placing deferred proposals somewhere in tree, rather than closing the PR would be more helpful. I don't imagine there will be huge number of these anyway and merging these PRs would likely be something that can be handled by an editorial team.

Reply:
How about a single `inactive` subdirectory, especially if we think there won't be a lot of these?